package.json

```diff
@@ -96,7 +96,9 @@
     ],
     "okapiInterfaces": {
       "eholdings": "4.0",
-      "tags": "1.0",
+      "tags": "1.0"
+    },
+    "optionalOkapiInterfaces": {
```

Comment:
Changing required to optional is a breaking change.

Reply:
it shouldn't be breaking as it's still backward compatible